### PR TITLE
9th upgrade of version to align with JBEAP 7.0.0

### PR DESCRIPTION
**Pull request by mbiarnes**
upgraded versions
org.slf4j
org.yaml.snakeyaml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -334,7 +334,8 @@
     <version.org.rhq>4.8.0</version.org.rhq>
     <!-- Avoid using scannotation in favor of its successor org.reflections -->
     <version.org.scannotation>1.0.3</version.org.scannotation>
-    <version.org.slf4j>1.7.2</version.org.slf4j>
+    <version.org.slf4j>1.7.7</version.org.slf4j>
+    <version.org.slf4j.jdk14>1.5.6</version.org.slf4j.jdk14>
```
Comment (psiroky) [inline pom.xml]: What was the reason to use this old version?

**Comment by mbiarnes** [comment]
@psiroky: this was downgraded by me since JBEAP 7.0.0 uses org.slf4j:slf4j-jdk14:1.5.6

**Comment by mbiarnes** [comment]
@psiroky should I reject this "old" version?

**Comment by psiroky** [comment]
From compatibility point of view, I think we should use the same version for both slf4j-api and slf4j-jdk14. So yes, please use just the 1.7.7 for both.

**Comment by hchiorean** [comment]
+1

**Comment by cunningt** [comment]
+1

**Comment by psiroky** [comment]
Merged as part of https://github.com/jboss-integration/jboss-integration-platform-bom/commit/f5ac8e34052f5d640327515242f41dd2c4dd08ad.

**Pull request closed without merging**